src/Vue/assets/package.json

```diff
@@ -49,7 +49,7 @@
         "@testing-library/jest-dom": "^6.6.3",
         "@testing-library/user-event": "^14.6.1",
         "@types/webpack-env": "^1.16",
-        "@vitejs/plugin-vue": "^4.4.0",
+        "@vitejs/plugin-vue": "^5.1.5",
```

Comment:
Can we use latest ^6.0 version instead https://github.com/vitejs/vite-plugin-vue/blob/plugin-vue@6.0.1/packages/plugin-vue/CHANGELOG.md#601-2025-07-28?